.cspell.json

```
@@ -1231,6 +1235,7 @@
     "pointcloud",
     "pointclouds",
     "pointpainting",
+    "PointPillars",
```

This is invalid.

```suggestion
```

---

Did you intend this?

```suggestion
    "pointpillars",
```

---

To clarify Miyake-san's point, i will add some explanation, for @Shin-kyoto .
`cspell` recognize capital letter as an word splitter, in CamelCase.
So, `PointPillars` will be split into `Point` and `Pillars`, and they will make no errors by cspell.
This is a reason Miyake-san said "This is invalid".

And, as Miyake-san suggested, "pointpillars" may be all you want.

---

Thank you for telling me that!
So I'll add `pointpillars` by workflow.